MonoGame.Framework.Content.Pipeline/Builder/PipelineBuildEvent.cs

```diff
@@ -122,7 +123,7 @@ public static PipelineBuildEvent Load(string filePath)
             PipelineBuildEvent pipelineEvent;
             try
             {
-                using (var textReader = new StreamReader(fullFilePath))
+                using (var textReader = new XmlTextReader(fullFilePath))
```

Comment:
Why does using XmlTextReader instead of Stream reader resolve this issue?

Reply:
With StreamReader 'FirstCharacter' was deserialized as empty string.
XmlTextReader preserves whitespaces in nodes and correctly deserialize it as " ".

<code>
  &lt;Parameters&gt;
    &lt;Key>FirstCharacter&lt;/Key&gt;
    &lt;Value>&nbsp;&lt;/Value&gt;
  &lt;/Parameters&gt;
</code>

Reply:
Makes sense.